qiita_pet/handlers/analysis_handlers.py

```diff
@@ -75,7 +76,7 @@ def post(self):
         name = self.get_argument('name')
         desc = self.get_argument('description')
         analysis = Analysis.create(self.current_user, name, desc,
-                                   from_default=True)
+                                   qiita_config.portal, from_default=True)
```

Comment:
If you are going to populate it anyways from the DB, better to have it already embedded on qiita db....